cluster-service/deploy/integration/cluster-service-namespace.yaml

```diff
@@ -72,3 +78,49 @@ objects:
       namespace: ${NAMESPACE}
     data:
       cs-client-id: ${CLIENT_ID}
+  - apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: ${ORPHANED_NAMESPACE_CLEANER}
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: orphaned-namespace-cleaner-cronjob
+      namespace: ${ORPHANED_NAMESPACE_CLEANER}
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: orphaned-namespace-cleaner-cronjob
+    subjects:
+      - kind: ServiceAccount
+        name: orphaned-namespace-cleaner-cronjob
+        namespace: ${ORPHANED_NAMESPACE_CLEANER}
+    roleRef:
+      kind: ClusterRole
+      name: namespace-admin
+      apiGroup: rbac.authorization.k8s.io
+  - apiVersion: batch/v1
+    kind: CronJob
+    metadata:
+      name: orphaned-namespace-cleaner
+      namespace: ${ORPHANED_NAMESPACE_CLEANER}
+    spec:
```

Comment:
What happens if for example there's another job already running? are we limiting the concurrency of the jobs?

Reply:
Already this job will run once a day. It is going to be very rare that a job pod will still be running.